sw/device/tests/BUILD

```diff
@@ -7,10 +7,10 @@
 # subdirectory name, because the build targets are really declared at the bottom
 # of this file, rather than in the subdirectories.
 #sw_tests = {
-  # 'test_name': {
-  #   'library': test_lib,
-  #   'dv_frames': true/false, # (can be omitted, defaults to `false`)
-  # },
+# 'test_name': {
+#   'library': test_lib,
+#   'dv_frames': true/false, # (can be omitted, defaults to `false`)
+# },
 #}
```

Comment:
This file seems to be all comments right now.  If these are tests meant to run on a riscv target, we should omit them for now.